<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManager</class>
 <widget class="QMainWindow" name="BookManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>iYandaBooks 图书管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/src.qrc">
    <normaloff>:/icon/icons/icons8-books-96.png</normaloff>:/icon/icons/icons8-books-96.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="action_filter"/>
    <addaction name="actionrefresh_filter"/>
    <addaction name="separator"/>
    <addaction name="actionadd_book"/>
    <addaction name="actiondelete_book"/>
    <addaction name="actionchange_book"/>
    <addaction name="separator"/>
    <addaction name="actionlogout"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>销售</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="actionpurchase"/>
    <addaction name="separator"/>
    <addaction name="actionstatistics"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_filter"/>
   <addaction name="actionrefresh_filter"/>
   <addaction name="separator"/>
   <addaction name="actionadd_book"/>
   <addaction name="actiondelete_book"/>
   <addaction name="actionchange_book"/>
   <addaction name="separator"/>
   <addaction name="action_3"/>
   <addaction name="actionpurchase"/>
   <addaction name="actionstatistics"/>
   <addaction name="separator"/>
   <addaction name="actionlogout"/>
   <addaction name="actionexit"/>
   <addaction name="separator"/>
   <addaction name="action_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionadd_book">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-plus-96.png</normaloff>:/icon/icons/icons8-plus-96.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加</string>
   </property>
  </action>
  <action name="actiondelete_book">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-remove-96.png</normaloff>:/icon/icons/icons8-remove-96.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionchange_book">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-edit-96.png</normaloff>:/icon/icons/icons8-edit-96.png</iconset>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-key-96.png</normaloff>:/icon/icons/icons8-key-96.png</iconset>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-about-96.png</normaloff>:/icon/icons/icons8-about-96.png</iconset>
   </property>
   <property name="text">
    <string>关于此应用</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-sale-96.png</normaloff>:/icon/icons/icons8-sale-96.png</iconset>
   </property>
   <property name="text">
    <string>销售</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-leave-96.png</normaloff>:/icon/icons/icons8-leave-96.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionpurchase">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-buy-96.png</normaloff>:/icon/icons/icons8-buy-96.png</iconset>
   </property>
   <property name="text">
    <string>购入</string>
   </property>
  </action>
  <action name="actionstatistics">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-align-bottom-96.png</normaloff>:/icon/icons/icons8-align-bottom-96.png</iconset>
   </property>
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
  <action name="action_filter">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-filter-96.png</normaloff>:/icon/icons/icons8-filter-96.png</iconset>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
   <property name="toolTip">
    <string>筛选</string>
   </property>
  </action>
  <action name="actionrefresh_filter">
   <property name="icon">
    <iconset resource="src/src.qrc">
     <normaloff>:/icon/icons/icons8-replay-96.png</normaloff>:/icon/icons/icons8-replay-96.png</iconset>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="toolTip">
    <string>重置</string>
   </property>
  </action>
  <action name="actiondisable_action_for_selected_row">
   <property name="text">
    <string>disable_action_for_selected_row</string>
   </property>
  </action>
  <action name="actionenable_action_for_selected_row">
   <property name="text">
    <string>enable_action_for_selected_row</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src/src.qrc"/>
 </resources>
 <connections/>
</ui>
